File: source/aws/py/lambda/lambda_entry.py
# use the '../.env' environment variables for local debugging
import use_env
from typing import Dict

def handler(event: Dict[str, any], context: Dict[str, any] = None):
    print('AWS Lambda Project "@{project}".')
    return {'status': 200}

if __name__ == '__main__':
    handler({})

File: source/aws/ts/lambda/lambda_entry.ts
import './use_env';
import {} from 'aws-sdk';

interface IEvent{
}

export async function handler (event: IEvent) {
    console.log('AWS Lambda Project "@{project}".');
    return {
        statusCode: 202,
        body: 'Accepted.'
    };
};

// Entry point fo the process
if (require.main == module) {
    (async () => await handler({}))();
}